lib/sb3_definitions.json

```diff
@@ -347,8 +347,7 @@
             "description": "Description of property (and/or property/value pairs) for sprites.",
             "properties": {
                 "name": {
-                    "type": "string",
-                    "not": {"enum": ["Stage", "stage"]}
+                    "type": "string"
```

Comment:
Should validation not fail if you try to name a sprite `_stage_`, or is that handled elsewhere?

```suggestion
                    "type": "string"
                    "not": {"enum": ["_stage_"]}
```

Reply:
@joker314, good catch! It looks like both the 2.0 and 3.0 editors prevent renaming a sprite to `_stage_` so that should be included in the validation.